.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,17 @@
+- id: vermin
+  name: Check minimum Python version
+  description: Checks the minimum needed Python version to run your package. Only runs on staged files (your target must be `x.y-` rather than `x.y` for it to work properly).
```

Comment:
This is great. In addition to mentioning that it's best to use `x.y-`, I think we should also mention that `--violations` is a good idea to use here since it works well when having only one target and it shows the violating values:
```
  --violations
        Show only results that violate versions described by --target arguments,
        which are required to be specified. Verbosity mode is automatically set to
        at least 2 in order to show violations in output text, but can be increased
        if necessary.

        If no rules are triggered while used in conjunction with --target, an exit
        code 0 will still be yielded due to inconclusivity.
```

An example of the above could be a file that doesn't do much and thus results in `~2 ~3` minimums, which means it cannot conclude anything concretely. But two targets of `~2` and `~3` differ from the input of `x.y-`. So that's a good idea to include.

Reply:
Oh, that's super handy, I didn't actually know about this argument!